etc/fdns.service

```diff
@@ -21,5 +21,98 @@ Restart=on-failure
 #    and change the service type from `forking' to `simple'.
 #StandardOutput=append:/tmp/fdns-log.txt
 
+AmbientCapabilities=
+LockPersonality=true
```

Comment:
The following options are unsupported on Ubuntu 16.04 LTS with systemd 229-4ubuntu21.23:

LockPersonality
RestrictNamespaces
RestrictRealtime
RestrictSUIDSGID
ProtectHostname
PrivateMounts

ProtectKernelTunables
ProtectKernelModules
ProtectControlGroups

Reply:
Is anything broken or just not supported? On Fedora 30 is `ProtectHostname` also not supported, but it is just logged and ignored.

Reply:
These are indeed unsupported but not breaking anything. Just log pollution. Thanks for the work on this BTW!